### ISSUE TITLE
Cloud you add account field to all the reports

### ISSUE THREAD

**Issue by sreeramsanjay** [capitalone/cloud-custodian#1521]
It would be very useful to have the account field name on reports.

**Comment by kapilt** [comment]
the tools/c7n-org tool is setup for multiple accounts and does account name in the reports.

**Comment by sreeramsanjay** [comment]
Thanks Kapil, this is very useful and is there any reference document for c7n-org usage.

**Comment by kapilt** [comment]
not yet, i've heard reports from a few users using that are happy with it, was hoping to get some contribs on docs.  the config file format is 'documented' via the jsonschema in the source. The cli builtin help covers most of the rest. definitely could use some more verbose docs and examples, here's a start.

```$ cat accounts.yml
accounts:
- account_id: '1111111111'
  name: myorg-account-b
  regions:
  - us-east-1
  role: arn:aws:iam::1111111111:role/Reporting-CrossAccount
  tags:
  - type:dev
  - division:business
  - partition:us
  - scope:normal
- account_id: '22222222'
  name: myorg-account-b
  regions:
  - us-east-1
  role: arn:aws:iam::22222222:role/Reporting-CrossAccount
  tags:
  - type:dev
  - division:itops
  - partition:us
  - scope:normal
```

here's running a cfn stack deploy across all the accounts tagged dev and partition:us, org runner supports effectively running arbitrary scripts across accounts as a bonus feature.

```
$ c7n-org run-script -s ~/waf-deploy -c accounts.yml -r us-east-1 \
                      -t "partition:us" \
                      -t "type:dev" \
                      aws cloudformation deploy \
                      --stack-name default-global-waf \
                      --template-file=deploy/waf-default.yml
```

and then running a named policy across the accounts in the config file
```
c7n-org run -s ~/out -c accounts.yml  -u all-resources.yml -p all-s3
```

and then reporting with division, account, and some resource tag extraction

```
$ c7n-org report -s ~/out -c accounts.yml -u all-resources.yml --format csv --field "Division=division" --field "Env=type" --field "App=tag:App" --field "Owner=tag:Owner" -p all-s3 -f ~/s3-buckets.csv
```

**Comment by sreeramsanjay** [comment]
Thank you,
I am getting below error on report.

Traceback (most recent call last):
  File "/root/custodian/bin/c7n-org", line 9, in <module>
    load_entry_point('c7n-org==0.0.1', 'console_scripts', 'c7n-org')()
  File "/root/custodian/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/root/custodian/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/root/custodian/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/root/custodian/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/root/custodian/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/root/custodian/local/lib/python2.7/site-packages/c7n_org/cli.py", line 220, in report
    fields=prefix_fields)
TypeError: __init__() got an unexpected keyword argument 'fields'

**Comment by kapilt** [comment]
hmm.. there seems to be a delta between the version in pypi and trunk, i'll cut a new release.

**Comment by kapilt** [comment]
uploaded 0.0.2

**Comment by sreeramsanjay** [comment]
Still same issue with c7n-org reports.

Traceback (most recent call last):
  File "/proj/custodian/bin/c7n-org", line 9, in <module>
    load_entry_point('c7n-org==0.0.2', 'console_scripts', 'c7n-org')()
  File "/proj/custodian/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/proj/custodian/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/proj/custodian/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/proj/custodian/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/proj/custodian/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/proj/custodian/local/lib/python2.7/site-packages/c7n_org/cli.py", line 228, in report
    fields=prefix_fields)
TypeError: __init__() got an unexpected keyword argument 'fields'
(custodian)[root@ip-172-28-0-210 proj]# pip list|grep c7n
c7n (0.8.26.0)
c7n-org (0.0.2)

**Comment by kapilt** [comment]
fwiw this was fixed in #1622 re c7n-org errors
